### ISSUE TITLE
Feature Request: For adding/removing extended file attributes

### ISSUE THREAD

**Issue by Rakesh777** [nvzqz/FileKit#58]
It would be great to have a way to add/remove extended file attributes. For example, If I wanted to associate different display name for file rather than original name of file. In such case, extended file attributes can come in handy.

**Comment by phimage** [comment]
Yep, currently the functions are private (_setAttribute and _setAttributes)
and all accessor are get only due to possible errors throwed
